presto-hive/pom.xml

```diff
@@ -171,6 +171,12 @@
         </dependency>
 
         <dependency>
+            <groupId>com.facebook.presto</groupId>
```

Comment:
@martint Since we need to get the version of current Presto, and this information is stored in `NodeVersion` which is defined in presto-client, to make this information injected and understood by Hive connector, presto-client is required.